src/GSoft.Extensions.Http.Authentication.ClientCredentialsGrant/ClientCredentialsTokenHttpMessageHandler.cs

```diff
@@ -27,27 +28,29 @@ internal sealed class ClientCredentialsTokenHttpMessageHandler : PolicyHttpMessa
 
     private readonly IClientCredentialsTokenManagementService _tokenManagementService;
     private readonly string _clientName;
+    private readonly ClientCredentialsOptions _clientCredentialsOptions;
 
-    public ClientCredentialsTokenHttpMessageHandler(IClientCredentialsTokenManagementService tokenManagementService, string clientName)
+    public ClientCredentialsTokenHttpMessageHandler(IClientCredentialsTokenManagementService tokenManagementService, string clientName, ClientCredentialsOptions clientClientCredentialsOptions)
```

Comment:
In the [rest of the code](https://github.com/search?q=repo%3Agsoft-inc%2Fgsoft-authentication-clientcredentialsgrant+%22options+%3D+%22&type=code) we name the `ClientCredentialsOptions` instances `options` or `_options` so we might want to do this here too (both param + field)